Pseudocode/Sorting/Quick Sort/SourceCode.tex

```diff
@@ -0,0 +1,66 @@
+% Set the Page Layout
+\documentclass[12pt]{article}
+\usepackage[inner = 2.0cm, outer = 2.0cm, top = 2.0cm, bottom = 2.0cm]{geometry}
+
+% Package to write pseudo-codes
+\usepackage{algorithm}
+
+% Don't Remove the 'end' at the end of the algorithm
+\usepackage{algpseudocode}
+
+% Manually remove the 'end' for some sections
+\algtext*{EndIf}
+\algtext*{EndFunction}
+\algtext*{EndFor}
+
+% Define Left Justified Comments
+\algnewcommand{\LeftComment}[1]{\Statex \(\triangleright\) #1}
+
+% Remove the Numbering of the Algorithm
+\usepackage{caption}
+\DeclareCaptionLabelFormat{algnonumber}{Algorithm}
+\captionsetup[algorithm]{labelformat = algnonumber}
+
+% Define the command for a boldface instructions
+\newcommand{\Is}{\textbf{ is }}
+\newcommand{\To}{\textbf{ to }}
+\newcommand{\Downto}{\textbf{ downto }}
+\newcommand{\Or}{\textbf{ or }}
+\newcommand{\And}{\textbf{ and }}
+% Use them inside Math-Mode (Hence the space!)
+
+\begin{document}
+
+\begin{algorithm}
+
+  \caption{Sort an array using Divide and Conquer approach}
+  
+  \begin{algorithmic}[1]
+    \Statex
+    
+    \Procedure{$Quick\_Sort$}{$arr, low, high$}
```

Comment:
The function names are quite small, hence we can just put them in all-Caps for this problem. To do that, just remove the math mode from the function's name. 

Moreover, the arguments looked cluttered together. To separate them, use space. To give a space inside a math mode, use `\ `. So, in this case it becomes `$arr, \ low, \ high$`